design-system/BottomSheet/BottomSheetBackdropOpacity.tsx

```diff
@@ -4,15 +4,29 @@ import {
 } from "@gorhom/bottom-sheet";
 import { memo } from "react";
 
+import { useAppTheme } from "../../theme/useAppTheme";
+
 export const BottomSheetBackdropOpacity = memo(function BackdropOpacity(
   props: GorhomBottomSheetBackdropProps
 ) {
+  const { style, animatedIndex, animatedPosition, ...rest } = props;
+
+  const { theme } = useAppTheme();
+
   return (
     <GorhomBottomSheetBackdrop
-      {...props}
       opacity={0.6}
       disappearsOnIndex={-1}
       appearsOnIndex={0}
+      animatedIndex={animatedIndex}
+      animatedPosition={animatedPosition}
+      style={[
+        {
+          backgroundColor: theme.colors.background.scrim,
+        },
```

Comment:
I've put your change at the design-system level because I think that's the styling we'll want to use for all our bottom sheet anyway